scripts/vet.sh

```diff
@@ -41,11 +41,9 @@ if [[ "$1" = "-install" ]]; then
   popd
   if [[ -z "${VET_SKIP_PROTO}" ]]; then
     if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
```

Comment:
This check is not required now. It would also be nice to install protoc using `./vet.sh -install`